his/core/application-integration-3270designer-1.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 3270 Design Tool
+description: Design green-screen applications using the 3270 Design Tool.
+author: haroldcampos
+ms.author: "hcampos"
```

Comment:
No need to include author metadata as you're listed in the cocfx.json file as the default author:

```suggestion
```